src-opam/dockerfile_windows.ml

```diff
@@ -75,16 +98,16 @@ module Cygwin = struct
     }
 
   let run_sh ?(cyg=default) fmt = ksprintf (run {|%s\bin\bash.exe --login -c "%s"|} cyg.root) fmt
+  let run_sh_ocaml_env ?(cyg=default) args fmt = ksprintf (run_sh "ocaml-env exec %s -- %s" args) fmt
```

Comment:
The syntax for this is weird (I can never remember it...)
```suggestion
  let run_sh_ocaml_env ?cyg:(_ = default) args fmt = ksprintf (run_sh "ocaml-env exec %s -- %s" args) fmt
```
that's how you specify a default value for an optional parameter which you then don't use without triggering warning 27 😜

Reply:
Weird, I did not see the warning trigger. Noted, I didn't know that syntax. I even learned the shortcut `?(arg=value)` here :)

Reply:
Oh, but it's a definitely a bug that the parameter is not used. It should be passed to the nested `run_sh` function.

Reply:
Ah, so it is - I didn't stare at it hard enough to spot that!